components/layouts/TheMenu/index.vue

```diff
@@ -98,6 +99,7 @@ import MenuMixin from '~/mixins/menu'
 @Component
 export default class TheMenu extends Mixins(MenuMixin) {
   isMobileMenuVisible: boolean = false
+  isCommunityDropdownVisible = false
```

Comment:
I did not find any use of this field in the code. Is it safe to remove completely?